docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -0,0 +1,103 @@
+---
+title: MSTest SDK
+author: MarcoRossignoli
+description: Overview of MSTest.Sdk.
+ms.author: mrossignoli
+ms.date: 02/13/2024
+---
+
+# MSTest SDK
+
+## Overview
+
+`MSTest.Sdk` is an [MSBuild project SDK](/visualstudio/msbuild/how-to-use-project-sdk) for building MSTest apps.  
+It's possible to build an MSTest app without this SDK, however, the MSTest SDK is:
+
+* Clear and coincise letting you to focus on your tests without the noise of the test infrastructure.
+* Simple to configure.
+* The recommended target for most users.
+
+How to use the `MSTest.Sdk` in a project:
+
+  ```xml
+  <!-- 
+       You can set a different sdk version, 3.3.0 is an example
+       You can set the SDK version at solution level using the global.json
+       Follow this guide:
+       https://learn.microsoft.com/visualstudio/msbuild/how-to-use-project-sdk?#how-project-sdks-are-resolved
+  -->
+  <Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+    </PropertyGroup>
+    <!-- references to the code to test -->      
+  </Project>
+  ```
+
+When you `build` the project all the needed components will be restored and installed using the standard NuGet workflow set by your project. Usually the components are downloaded from the official NuGet feed unless set differently.
+
+You don't need anything else to build and run your tests and you can use the same tooling (i.e. `dotnet test`, Visual Studio etc...) used by a ["classic"](./index.md/#mstest) MSTest project.
+
+## `MSTest.Sdk` with the `MSTest runner`
+
+The `MSTest.Sdk` supports the [`MSTest runner`](./unit-testing-mstest-runner-intro.md) and simplify the configuration of extensions to make the experience easy and smooth.
+
+How to use the `MSTest.Sdk` with the `MSTest runner` in a project:
+
+  ```xml
+  <Project Sdk="MSTest.Sdk/3.3.0">
+    <PropertyGroup> 
+        <TargetFramework>net8.0</TargetFramework>
+        <EnableMSTestRunner>true</EnableMSTestRunner> 
+        <OutputType>Exe</OutputType> 
```

Comment:
I think this should be the default when using the SDK but let's bring it up to discussion.

Reply:
Do we have enough users/data to start with the runner in 3.3.0?

Reply:
discussed off line